OrbitGl/ModulesDataView.cpp

```diff
@@ -38,23 +38,23 @@ const std::vector<DataView::Column>& ModulesDataView::GetColumns() {
 
 //-----------------------------------------------------------------------------
 std::string ModulesDataView::GetValue(int row, int col) {
-  const std::shared_ptr<Module>& module = GetModule(row);
+  const ModuleData* module = GetModule(row);
 
   switch (col) {
     case COLUMN_INDEX:
       return std::to_string(row);
     case COLUMN_NAME:
-      return module->m_Name;
+      return module->name();
     case COLUMN_PATH:
-      return module->m_FullName;
+      return module->path();
     case COLUMN_ADDRESS_RANGE:
-      return module->m_AddressRange;
+      return module->address_range();
     case COLUMN_HAS_PDB:
-      return module->IsLoadable() ? "*" : "";
+      return "*";
     case COLUMN_PDB_SIZE:
```

Comment:
While you are at it, should those columns maybe be renamed? Here and in the UI. The naming Pdb might be confusing in a Linux context. Also, maybe remove the columns `COLUMN_HAS_PDB` completely

Reply:
This could also be a different PR

Reply:
okay